.travis.yml

```
@@ -24,9 +24,12 @@ notifications:
 
 jobs:
   include:
+  - stage: format
+  - name: Detekt
+    script: "./detekt"
   - stage: test
     name: Lint
-    script: "./gradlew lintRelease"
+    script: "./gradlew lintRelease"z
```

Is this a problem? The trailing `z`
```suggestion
    script: "./gradlew lintRelease"
```